### PR TITLE
Improve the CI speed

### PR DESCRIPTION
**Pull request by bam241**
this should improve CI speed by removing installation of `git` and `openssh` in the docker image

### CODE REVIEW

**Comment by bam241** [comment]
(this might not work until the cyclus-deps docker gets updated)

**Comment by gonuke** [comment]
Does this need a rebase?  It looks good to me (since it's passing), but I'm not sure how we ensure that the docker image has git now?

**Comment by bam241** [comment]
> Does this need a rebase? It looks good to me (since it's passing), but I'm not sure how we ensure that the docker image has git now?

it was already there (otherwise CI would have failed). but I added it to the apt-get in the docker file just to be sure it will work in the future....

File: news/faster_ci.rst [hunk 1]
```diff
@@ -0,0 +1,12 @@
+**Added:** None
+
+**Changed:** None
+-  git and open-ssh added to the dockerfile (removed from CI)
+
+**Deprecated:** None
+
+**Removed:** None
+-  git and open-ssh form CI (added to the dockerfile)
```
Comment (gonuke) [inline news/faster_ci.rst]:
```suggestion
-  git and open-ssh from CI (added to the dockerfile)
```

**Comment by gonuke** [comment]
Thanks @bam241

**Pull request merged**